agent/taskresource/cgroup/cgroup_unsupported.go

```diff
@@ -0,0 +1,148 @@
+// +build !linux
+// Copyright 2014-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cgroup
+
+import (
+	"sync"
+	"time"
+
+	"github.com/aws/amazon-ecs-agent/agent/taskresource"
+)
+
+const (
+	resourceName = "cgroup"
+)
+
+// CgroupResource represents Cgroup resource
+type CgroupResource struct {
+	desiredStatusUnsafe taskresource.ResourceStatus
```

Comment:
Why do you need any members at all in this struct? Everything should be a no-op, right?

Reply:
I just left the fields there since this is used in tests (more like a fake resource). so only having the status fields there and not actual implementations like `Create` or `Cleanup`.

Reply:
ok, what do you think about adding a new struct called TestResource and using that for testing? we cannot really use the mock resource for tests, since they test the states progression.